pages/views/_includes/landing/blogs.njk

```diff
@@ -0,0 +1,16 @@
+<div class="blog container">
+  <h2>{{ landing.blogs.title }}</h2>
+  <esl-animate target="::next::child" group></esl-animate>
+  <ul class="blog-container">
+    {% for item in landing.blogs.items | sortByDate | limit(3) %}
+      <li class="blog-item esl-animate-slide-left">
+        <div class="blog-content">
+          <div class="blog-content-name">{{ item.name }}</div>
+          <p class="blog-content-text">{{ item.text }}</p>
```

Comment:
Request to make text smaller and also please make sure that we have enough space between columns